ftw/solr/browser/resources/searchpage.scss

```diff
@@ -115,17 +115,26 @@
 }
 
 #selected-facets {
-  @include list();
+  @include list-horizontal();
 
   > li {
     @include list-group-item();
+    border-right: 1px solid $border-color-list-group;
+
+    &:last-child {
+      border-right: 0;
+    }
 
     > .facetRemoveItem {
       @extend .fa-icon;
       @extend .fa-remove;
       padding: 0;
-      display: inline;
+      display: inline-block;
       color: $color-danger;
+      overflow: hidden;
+      width: 1.5em;
+      height: 1.2em;
```

Comment:
@bierik Updated, thanks.

But the font-size needs to be in PX, since the parent (a-Tag) has a font-size of 0.